packages/bindings-test/src/main.rs

```diff
@@ -1,14 +1,15 @@
 extern crate core;
 
 mod chain_communication;
+
 mod consts;
 mod models;
-mod posts;
-mod profiles;
-mod reactions;
-mod relationships;
-mod reports;
-mod subspaces;
+
+#[cfg(feature = "legacy")]
+mod legacy;
+
+#[cfg(feature = "stargate")]
+mod stargate;
```

Comment:
stargate tests follows the legacy tests process, only types are different from the legacy one.